OSDOCS-10891:dedicated troubleshooting section for data cleanup (#80106)
openshift/openshift-docs: modules/microshift-data-cleaning-overview.adoc

// Module included in the following assemblies:
//
// * microshift_troubleshooting/microshift-cleanup-data.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-data-cleaning-overview_{context}"]
= Data cleaning overview

> The following command cleans all the {microshift-short} services and their data.

Author: I have removed this line.

> See the usage and available options of the `microshift-cleanup-data` script by running the script.

Reviewer: ```suggestion
See the usage and available options of the `microshift-cleanup-data` script by running the script without arguments.
```

Author: done

> See the usage and available options of the `microshift-cleanup-data` script by running the script without arguments.

Reviewer 2: ```suggestion
You can list the usage and available options of the `microshift-cleanup-data` script by running the script without arguments. Running the script without arguments does not cleanup any data or stop the {microshift-short} service.
```

.Procedure

> . Run the following script.

Reviewer 2: same

Author: done

Reviewer 2: Please update the command per newest review comments. TY!

> . Run the following script.

Reviewer 2: This is not docs style and must be updated. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command

> . Run the following script.

Reviewer 2: ```suggestion
. Run the following script.
```
List the usage and available options of the `microshift-cleanup-data` script by running the following command:

+
[source,terminal]
----

> $ sudo microshift-cleanup-data

Author: done

> You can list the usage and available options of the `microshift-cleanup-data` script by running the script without arguments. Running the script without arguments does not cleanup any data or stop the {microshift-short} service.

Reviewer: ```suggestion
You can see the usage and list available options of the `microshift-cleanup-data` script by running the script without arguments. Running the script without arguments does not delete any data or stop the {microshift-short} service.
```

> Some of the options in the following script operations are destructive and can cause data loss.

Reviewer 2: ```suggestion
Some of the options in the following script operations are destructive and can cause data loss. See the procedure for each argument for warnings.
```

> . List the usage and available options of the microshift-cleanup-data script by entering the following command:

Reviewer: Same as above?

See the usage and list the available...
